mne/decoding/tests/test_time_gen.py

```diff
@@ -296,18 +337,18 @@ def test_generalization_across_time():
     y[len(y) // 2:] += 2
     ys = (y, y + 1000)
     # Univariate and multivariate prediction
-    svc = SVC(C=1, kernel='linear')
+    svc = SVC(C=1, kernel='linear', probability=True)
+    reg = KernelRidge()
 
-    class SVC_proba(SVC):
-        def predict(self, x):
-            probas = super(SVC_proba, self).predict_proba(x)
-            return probas[:, 0]
+    def scorer_proba(y_true, y_pred):
+        roc_auc_score(y_true, y_pred[:, 0])
```

Comment:
It feels a bit strange to define the scorer for tests only, when users will be defining it itself later on ... but that's not for this PR.